sulis_editor/src/actor_creator_window.rs

```diff
@@ -196,13 +196,10 @@ impl ActorCreatorWindow {
                 .add_callback(Callback::new(Rc::new(move |widget, _| {
                     let (_, window) = Widget::parent_mut::<ActorCreatorWindow>(widget);
 
-                    let index = window.selected_images.get(&layer).unwrap().0;
-                    if index > 0 {
-                        let index = index - 1;
-                        window
-                            .selected_images
-                            .insert(layer, (index, Rc::clone(&images_ref[index])));
-                    }
+                    let index = (window.selected_images.get(&layer).unwrap().0 - 1) % len;
```

Comment:
Because `window.selected_images[&layer].0` is a usize, this will cause problems when it is 0.  In particular, it panics when running in debug mode, and wraps around in release mode.  Even when wrapping, I don't think the wrap around will be to the last entry either, but rather will be what seems like a random entry in the array.

I think the safest bet is to manually check for 0 and set it to `len - 1` in this case.